core/db/curibio/alembic/versions/c1ae9bb9fa14_pulse3_0_25_4.py

```diff
@@ -0,0 +1,24 @@
+"""pulse3 0.25.4
+
+Revision ID: c1ae9bb9fa14
+Revises: 871f6d005d86
+Create Date: 2022-09-23 10:57:43.588260
+
+"""
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = "c1ae9bb9fa14"
+down_revision = "871f6d005d86"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    query = "INSERT INTO pulse3d_versions (version) VALUES ('0.25.4')"
+    op.execute(query)
+
+
+def downgrade():
```

Comment:
You'll need to define the downgrade too. In this case it should just remove the row that `upgrade` adds